compiler/ir.py

```diff
@@ -960,7 +961,7 @@ def introduce_aggregators_for_consec_chunks(self, fileIdGen, in_aggregator_ids,
                 # TODO: turn node into cmd_invocation_with_io_vars since this is the only thing required in this function
                 self.create_generic_aggregator_tree(original_cmd_invocation_with_io_vars, parallelizer, map_in_aggregator_ids, out_aggregator_id, fileIdGen)
             else:
-                raise Exception("aggregator kind not yet implemented")
+                raise UnparallelizableError("aggregator kind not yet implemented")
```

Comment:
Maybe add more information in this message. For example, which command does this refer to?